lib/node_modules/@stdlib/string/prev-grapheme-cluster-break/lib/main.js

```diff
@@ -71,6 +71,12 @@ function prevGraphemeClusterBreak( str, fromIndex ) {
 	var cp;
 	var i;
 
+	var regexEmoji = new RegExp("\\p{Extended_Pictographic}", "ug");
+	var regexEmojiMatching = str.match(regexEmoji)
```

Comment:
You assume that `str` is a string here. But that assumption is not verified until L80.